crypto/asn1/asn_mime.c

```diff
@@ -516,6 +516,12 @@ int SMIME_crlf_copy(BIO *in, BIO *out, int flags)
     int len;
     char linebuf[MAX_SMLEN];
     int ret;
+
+    if (in == NULL || out == NULL) {
+        ERR_raise(ERR_LIB_ASN1, ERR_R_PASSED_NULL_PARAMETER);
```

Comment:
BTW: I always wondered why the result of BIO_write/BIO_printf etc. is not checked in this function

Reply:
Nice to hear.
Then how about giving your committer's approval?

Reply:
Yes, I'll do that if you add the error handling for the write functions.